src/main/java/com/renatusnetwork/momentum/storage/mysql/TablesDB.java

```diff
@@ -965,6 +968,46 @@ private static void createBankBidsKeys()
         DatabaseQueries.runQuery(foreignKeyQuery);
     }
 
+    private static void createCommandSigns() {
+        String query =
+                "CREATE TABLE " + DatabaseManager.COMMAND_SIGNS + "(" +
+                    "name VARCHAR(20) NOT NULL, " +
+                    "command VARCHAR(100), " +
+                    "world VARCHAR(30) NOT NULL, " +
+                    "x INT NOT NULL, " +
+                    "y INT NOT NULL, " +
+                    "z INT NOT NULL, " +
+                    "PRIMARY KEY (name), " +
+                    "INDEX location_index (world, x, y, z)" +
```

Comment:
Is this index needed? I'm trying to think of a case of where we would search world, x, y, z in db frequently

Reply:
its not needed i forgot to remove it when creating the sign name. will remove.